docs: Add a Japanese translation style guide. (#24735)
zulip/zulip: docs/translating/japanese.md

# Japanese translation style guide

## Rules

- Use full-width parentheses `（）` for half-width parentheses `()` in original text.

## Terms

- alert word - **キーワード**/**キーワード通知**

> "Alert" can be literally translated as "アラート", however, it gives a negative impression in Japanese. To describe the function clearly, use "キーワード" for "alert word (noun)" and "キーワード通知" for "alert word (verb)".

Reviewer: ```suggestion
  "Alert" can be literally translated as "アラート", however, it gives a negative impression in Japanese. To describe the function clearly, use "キーワード" for "alert word (noun)" and "キーワード通知" for "alert word (verb)".
```

>   "Alert" can be literally translated as "アラート", however, it gives a negative impression in Japanese. To describe the function clearly, use "キーワード" for "alert word (noun)" and "キーワード通知" for "alert word (verb)".

Reviewer 2: This distinction (noun vs. verb) doesn't sound right to me, because the English phrase "alert word" is always a noun, not a verb.

Looking at the existing translations, it seems like perhaps "キーワード通知" is used when the term has to stand alone to make clear what it's about (for example, in the navigation list at the left of the overall settings panel), while "キーワード" is used when the context helps fill in what it's about.

Perhaps this would be an accurate way to describe it?

```suggestion
  "Alert" can be literally translated as "アラート", however, it gives a negative impression in Japanese. To describe the function clearly, use "キーワード" or more fully "キーワード通知".
```
